### DISCUSSION TITLE
How does BERTopic do interpretation?

### DISCUSSION THREAD

**Discussion by Mitraic** [MaartenGr/BERTopic#2155] [General]
What does the output for interpretation look like?
Is there any function to do topic interpretation?

**Comment by MaartenGr** [comment]
Could you define "topic interpretation"? As it is currently implemented, you do have a [bunch of representation models](https://maartengr.github.io/BERTopic/getting_started/representation/llm.html) that will give you all kinds of different topic representations for you to interpret.
